scripts/examples/bash-cli/README

```diff
@@ -1,49 +1,28 @@
-
-Introduction
-
-   This file describes the XE CLI + bash examples included in the [1]Software
-   Development Kit (SDK) for the [2]XenServer 4.0 product family.
-
 XE CLI + bash examples
 
-  install-debian
-
-   [3][ browse ] [4][ download ]
+ [install-debian][1]
```

Comment:
It's markdown recognisable by github if the file has .md extension - which I've just realised this file does not have, so thanks for pointing it out, I'll rename it.